Traceback in integration tests:

https://app.shippable.com/github/ansible/ansible/runs/102832/53/console

```
23:38 Traceback (most recent call last):
23:38   File "/root/ansible/bin/ansible-playbook", line 110, in <module>
23:38     exit_code = cli.run()
23:38   File "/root/ansible/lib/ansible/cli/playbook.py", line 111, in run
23:38     results = pbex.run()
23:38   File "/root/ansible/lib/ansible/executor/playbook_executor.py", line 120, in run
23:38     if vname not in self._variable_manager.extra_vars:
23:38 AttributeError: 'VariableManager' object has no attribute 'extra_vars'
```

--- a/lib/ansible/vars/manager.py
+++ b/lib/ansible/vars/manager.py
@@ -138,6 +138,10 @@ class VariableManager:
         self._options_vars = data.get('options_vars', dict())
         self.safe_basedir = data.get('safe_basedir', False)
 
+    @property
+    def extra_vars(self):
+        return self._extra_vars
+
     def set_inventory(self, inventory):
         self._inventory = inventory
 

